dist/index.html

```diff
@@ -1 +1,27 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Wbpack Exercise</title><script defer="defer" src="main.js"></script></head><body><h1>Hello webpack!</h1></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <title>Lemonade</title>
```

Comment:
The 1st line come from command npm run build - the right one. But I can see, that you delete it and copy paste the code from index.html from folder "src". Please, consider, that is folder "dist" the html file should crated automatically after npm run build command